samples/BenchmarkDotNet.Samples.FSharp/BenchmarkDotNet.Samples.FSharp.fsproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net462;net8.0</TargetFrameworks>
     <GenerateProgramFile>false</GenerateProgramFile>
+    <!-- Disable parallel tests between TargetFrameworks -->
+    <TestTfmsInParallel>false</TestTfmsInParallel>
```

Comment:
Is this necessary if it's already set from `BenchmarkDotNet.TestAdapter.props`?

Reply:
Oh, it's necessary because it's not a package reference, but a project reference.